src/ffi/modsupport.rs

```diff
@@ -53,7 +53,13 @@ extern "C" {
     // skipped non-limited _PyArg_UnpackKeywords
     // skipped non-limited _PyArg_Fini
 
-    // skipped PyModule_AddObjectRef
+    #[cfg(Py_3_10)]
+    #[cfg_attr(docsrs, doc(cfg(Py_3_10)))]
+    pub fn PyModule_AddObjectRef(
+        arg1: *mut PyObject,
+        arg2: *const c_char,
+        arg3: *mut PyObject,
```

Comment:
```suggestion
        module: *mut PyObject,
        name: *const c_char,
        value: *mut PyObject,
```

That's the names these have in the documentation